lib/core/src/lib/info-drawer/info-drawer.component.scss

```diff
@@ -1,12 +1,23 @@
 .adf {
     &-info-drawer {
-        display: block;
+
+        .adf-metadata-properties-panel{
+            display:block;
+            margin:0;
```

Comment:
Add a space. In the styles, the format should be 'property:<sapce>value'.

Reply:
Done